feature/todo-list/src/main/kotlin/pl/mobite/todoapp/todolist/ui/TodoListViewModel.kt

```diff
@@ -65,6 +68,7 @@ class TodoListViewModel(
         reduce { withProgress() }
         val updatedItem = item.copy(isDone = isDone)
         updateTodoItemUseCase(updatedItem)
+        sideEffect(ItemUpdatedSideEffect)
```

Comment:
Pure naming style: maybe having action in function name can give it more readability, so similar to `processAction` have `sendSideEffect`.

Reply:
[e01ab62](https://github.com/mdabrowski89/TodoApp/pull/5/commits/e01ab626cffb2891a00c1cd44822f6671cc5a1a6)